ansible/roles/host-ocp4-provisioner/tasks/ec2_prereqs.yml

```diff
@@ -0,0 +1,63 @@
+- name: Get awscli bundle
+  get_url:
+    url: https://s3.amazonaws.com/aws-cli/awscli-bundle.zip
+    dest: /tmp/awscli-bundle.zip
+
+- name: Unzip awscli-bundle.zip
+  unarchive:
+    src: /tmp/awscli-bundle.zip
+    dest: /tmp/
+    remote_src: yes
+
+- name: Install awscli
+  become: yes
+  command: /tmp/awscli-bundle/install -i /usr/local/aws -b /bin/aws
+  args:
+    creates: /usr/local/aws
+
+- name: cleanup archive and tmp files
+  file:
+    path: "{{ item }}"
+    state: absent
+  loop:
+    - /tmp/awscli-bundle
+    - /tmp/awscli-bundle.zip
+
+- name: Create .aws directory
+  become: no
+  file:
+    path: ~/.aws
+    state: directory
+
+- name: Add aws credentials for sandboxes
```

Comment:
I may have missed it, but do we get rid of these after the install? There is really no reason to leave credentials on the bastion after the install is done since the cloud-credentials operator has used those and created what it needs inside of OpenShift.

Same for the following task, especially because it is a bigger account.

Reply:
We don't. If we do remove then uninstalling openshift will be a problem. But a problem we could potentially solve.